conformance/utils/http/http.go

```diff
@@ -135,6 +137,25 @@ func MakeRequest(t *testing.T, expected *ExpectedResponse, gwAddr, protocol, sch
 	return req
 }
 
+// calculateHost will calculate the Host header as per [HTTP spec]. To
+// summarize, host will not include any port if it is implied from the scheme. In
+// case of any error, the input gwAddr will be returned as the default.
+//
+// [HTTP spec]: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.23
+func calculateHost(gwAddr, scheme string) string {
+	host, port, err := net.SplitHostPort(gwAddr)
+	if err != nil {
+		return gwAddr
+	}
+	if strings.ToLower(scheme) == "http" && port == "80" {
```

Comment:
shouldnt this be OR ?

Reply:
sure its fine from a request URL perspective, wanted to make sure that this field is not being used to compare response Location header, guess im mixing this with https://github.com/kubernetes-sigs/gateway-api/pull/1980 which I shouldnt, my bad, resolving, thanks